internal/output/plain.go

```diff
@@ -324,3 +330,30 @@ func trimValue(value string, size int) string {
 	}
 	return value
 }
+
+func shiftColsVal(opts []string) int {
+	for _, opt := range opts {
+		a := strings.TrimPrefix(opt, string(ShiftColsPrefix))
+		if len(a) < len(opt) {
+			n, err := strconv.Atoi(a)
+			if err != nil {
+				logging.Errorf("Cannot get shiftCols value: %v", err)
```

Comment:
```suggestion
				logging.Errorf("Cannot get shiftCols value: %s, error: %v", a, err)
```

Reply:
This was what was done initially, but the output was redundant with the strconv.Atoi error message. Is it OK to leave as-is?

Reply:
This is nitpicky at best, but I prefer my errors not to make any assumptions about what they are wrapping.